client/src/ui/css/ResultsDisplay.css

```diff
@@ -6,57 +6,194 @@
 
 .filter-container {
   font-family: "Source Sans Pro", sans-serif;
-  padding-left: 6%;
-  padding-top: 28px;
   line-height: 37px;
-  min-width: 200px;
 }
 
-.filter-title {
-  font-family: "Source Sans Pro", sans-serif;
-  font-size: 18px;
-  font-weight: var(--semi-bold-weight);
+@media (min-width: 1000px) {
+  .filter-container {
+    padding-left: 6%;
+    padding-top: 28px;
+    min-width: 200px;
+  }
+}
+
+@media (max-width: 1000px) {
+  .filter-container {
+    margin-top: 36px;
+    margin-left: 24px;
+    margin-right: 24px;
+  }
+}
```

Comment:
You can list multiple css properties inside one `@media` block. So instead of creating new media blocks for every property, create 2 (one for max-width, one for min-width) and put stuff in there.